.dependabot/config.yaml

```diff
@@ -0,0 +1,7 @@
+version: 1
```

Comment:
Silly question, is it okay that it's not `config.yml`? I don't know how sensitive the tooling is to the extension

Reply:
Not a silly question, just an oversight on my part, thanks.
I don't know if it would work or not actually.